hw/ip/spi_device/doc/_index.md

```diff
@@ -301,12 +302,20 @@ The module stays in the wait state until the read FIFO is not empty.
 The module sends `START` at the next byte when the logic sees the notempty signal of the read FIFO.
 Then the module pops data from the read FIFO and sends the data over SPI.
 
+The TPM submodule accepts the payload for the TPM write command without the `WAIT` state if the write FIFO is empty.
+In other case, the TPM submodule sends `WAIT` until the write FIFO becomes available (empty).
+
+### Configuring Return-by-HW registers
+
 The return-by-HW register values come from the SW read-writable CSRs.
 The module latches the CSRs from the SYS_CLK domain into the SPI SCK domain when CSb is asserted.
 The SW is allowed to modify the return-by-HW registers when CSb is not active.
 
-The TPM submodule accepts the payload for the TPM write command without the `WAIT` state if the write FIFO is empty.
-In other case, the TPM submodule sends `WAIT` until the write FIFO becomes available (empty).
+The [TCG PC Client Platform TPM Profile][TPM PCCP] spec describes in the section 6 that the TPM device returns registers values based on the received locality (address[15:12]) and the `TPM_ACCESS_x.activeLocality`.
+The HW uses `TPM_ACCESS_x.activeLocaltiy` and the address bit 15:12 to determine what value the logic should return.
+If `invalid_ool` configuration is set, the logic returns `INVALID` value to the host system, when the host system sends a read request to the Locality greater than 4.
```

Comment:
sorry, what is `invalid_ool`

Reply:
That's typo. Thanks for finding it. should be `invalid_locality`.